feature-libs/product-configurator/rulebased/occ/variant/variant-configurator-occ.adapter.spec.ts

```diff
@@ -119,18 +168,17 @@ describe('OccConfigurationVariantAdapter', () => {
 
     expect(mockReq.cancelled).toBeFalsy();
     expect(mockReq.request.responseType).toEqual('json');
-    expect(converterService.pipeable).toHaveBeenCalledWith(
-      VARIANT_CONFIGURATOR_NORMALIZER
-    );
-    mockReq.flush(productConfiguration);
+    mockReq.flush(productConfigurationOcc);
   });
 
   it('should call readConfiguration endpoint', (done) => {
     spyOn(converterService, 'pipeable').and.callThrough();
     occConfiguratorVariantAdapter
       .readConfiguration(configId, groupId, productConfiguration.owner)
-      .subscribe();
-    done();
+      .subscribe((resultConfiguration) => {
+        expect(resultConfiguration.configId).toEqual(configId);
+        done();
```

Comment:
Actually the done() function is not even needed here at all, as this test is strictly synchronous, because the flush method of the HTTP mock will emit immediately and synchronously.
However it is also not wrong here, as the subscribe block is really the last being exec